### PR TITLE
move source files for building fonts into this repo

### LINKED ISSUE #4: move makeFF and dependent files into this repo
That way we only have to update this repo and the main KaTeX repo when updating fonts instead of having to update three repos.  makeFF can be found in https://github.com/Khan/MathJax-dev.

### PR DESCRIPTION
**Pull request by kevinbarabash**
Having the source files in the same repo as the font files they generate should make updating the fonts easier.  To generate the fonts I ran `./buildFonts.sh`.  This fixes #4.

### CODE REVIEW

File: buildFonts.sh [hunk 1]
```diff
@@ -119,7 +74,7 @@ if [[ $(docker images "$IMAGE" | wc -l) -lt 2 ]]; then
 fi
 
 CMDS="set -ex
-test -f MathJax-dev.tar.gz || wget -O MathJax-dev.tar.gz '${URL}'
+test -f MathJax-dev.tar.gz
 mk=\$(tar tf MathJax-dev.tar.gz | grep 'fonts/OTF/TeX/Makefile\$')
 tar xf MathJax-dev.tar.gz
```
Comment (edemaine) [inline buildFonts.sh]: Shouldn't lines 77-79 (in new file) be removed?

Comment (kevinbarabash) [reply buildFonts.sh]: I'll give that a try.  I was trying to minimize the number of changes but if we can get rid of one of the tar/untar steps that would be good.

File: buildFonts.sh [hunk 2]
```diff
@@ -119,10 +74,8 @@ if [[ $(docker images "$IMAGE" | wc -l) -lt 2 ]]; then
 fi
 
 CMDS="set -ex
-test -f MathJax-dev.tar.gz || wget -O MathJax-dev.tar.gz '${URL}'
-mk=\$(tar tf MathJax-dev.tar.gz | grep 'fonts/OTF/TeX/Makefile\$')
+mk=fonts/OTF/TeX/Makefile
 tar xf MathJax-dev.tar.gz
-cd \"\${mk%fonts/*}\"
```
Comment (kevinbarabash) [inline buildFonts.sh]: This wasn't changing the directory so I removed it.

File: buildFonts.sh [hunk 3]
```diff
@@ -119,10 +74,7 @@ if [[ $(docker images "$IMAGE" | wc -l) -lt 2 ]]; then
 fi
 
 CMDS="set -ex
-test -f MathJax-dev.tar.gz || wget -O MathJax-dev.tar.gz '${URL}'
-mk=\$(tar tf MathJax-dev.tar.gz | grep 'fonts/OTF/TeX/Makefile\$')
 tar xf MathJax-dev.tar.gz
```
Comment (kevinbarabash) [inline buildFonts.sh]: Unpacking the `tar` file is still necessary.  In the future we might want to look at mounting the `src/` folder somewhere inside the docker.  I've had troubles mounting folders on OS X in the past that's why I'm avoiding it at the moment.

**Comment by kevinbarabash** [comment]
> I assume we're giving up the possibility of taking a newer version from MathJax (at least without more work), but if we're diverging anyway I assume that's OK. And if we were already using a modified repo, this definitely seems better.

We've already diverged and I think we'll probably continue to do so, especially with our preference for non-combining accent characters.

File: buildFonts.sh (hunk 3 above)
Comment (edemaine) [reply buildFonts.sh]: Ah, I see, that's within the Docker.  Makes sense.

**Pull request merged**